plugins/modules/aci_epg_to_contract.py

```diff
@@ -81,10 +83,11 @@
 - module: cisco.aci.aci_epg
 - module: cisco.aci.aci_contract
 - name: APIC Management Information Model reference
-  description: More information about the internal APIC classes B(fv:RsCons) and B(fv:RsProv).
+  description: More information about the internal APIC classes B(fv:RsCons), B(fv:RsProv), B(fv:RsProtBy, B(fv:RsConsIf, and B(fv:RsIntraEpg).
```

Comment:
missing the last ) in `B(fv:RsProtBy, B(fv:RsConsIf,`

Reply:
changed